opencadd/structure/superposition/engines/theseus.py

```diff
@@ -39,24 +39,41 @@ class TheseusAligner(BaseAligner):
 
     Parameters
     ----------
+    sequence_alignment : str, optional, default=MUSCLE
+        What algorithm will be used to calculate the
+        sequence alignment. Choose between:
+        - "MUSCLE"
+        - "CLUSTALO" (CLUSTAL OMEGA)
     alignment_max_iterations : int
-        number of iterations for alignment program (muscle)
+        number of iterations for alignment program (only for muscle)
```

Comment:
Why only for `muscle`? Could you add a comment to the docstring, please?

Reply:
In this `theseus.py` file, why mention muscle?

Reply:
`muscle` is called in this file. It is used for the sequence alignment.